### DISCUSSION TITLE
Follow-up missake

### DISCUSSION THREAD

**Discussion by iamevie** [monkeytypegame/monkeytype#4535] [Ideas]
Request to implement support for follow-up mistake. For example, take this string "I love ducks" if I would type "I loveducks" everything after love would be a mistake, even though ducks is spelled correctly. Ducks would be a follow-up mistake.

I'd like to see these as highlighted differently :)

**Comment by Miodec** [comment]
Im not exactly sure what you mean, all though stop on error: word, might be simmilar.

**Reply by iamevie** [reply]
oh, I'm sorry, let me try to explain it again.

If I have this text to type:

"I love typing as much as I love ducks" and I type this: 

"Ilove typing as much as I love ducks" then everything after the first I is marked as an error. This is correct but technical, the only error is a missing space between 'I' and 'love'. I think in such cases, all the follow-ups shouldn't mark as an error with the same weight as a misspelling. It should be a follow-up mistake. An error that is only one, as u messed up earlier, but else wouldn't be.

**Reply by Miodec** [reply]
Ah, i see. Well, thing like this would need to be done after the test was finished, and if a user sees 5 errors in a row they will most likely just abandon the test. Plus, this invites complexity and possible bugs. And lastly, it benefits inaccuracy (or rather, lowers the punishment) - my mentality is that i want to punish inaccuracy as much as possible, and award accuracy. Another thing that just came to my mind that a user could just stop pressing space (or only press it every 5 or so words) and still get a somehow good score, which I also don't like.

I fully understand that a word shifted one letter to the side is *technically* one error, but I don't think the pros outweigh the cons.

**Reply by iamevie** [reply]
Oh I wouldn't let it affect the score. It would just be nice to see it afterward :)
